### PR TITLE
[Balance] Fix Baton Pass TM learnset

### PR DESCRIPTION
**Pull request by EmberCM**
## What are the changes?
Added and removed some Pokemon from [Baton Pass TM](https://bulbapedia.bulbagarden.net/wiki/Baton_Pass_(move)) learnset

## Why am I doing these changes?
Someone asked for it in [Discord](https://discord.com/channels/1125469663833370665/1262618712674926704)

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)

### CODE REVIEW

File: src/data/tms.ts [hunk 1]
```diff
@@ -29332,7 +29332,11 @@ export const tmSpecies: TmSpecies = {
     Species.POLIWRATH,
     Species.RAPIDASH,
```
Comment (Tempo-anon) [inline src/data/tms.ts:29333]: Just a quick check reveals Rapidash also lost access to baton pass via tm in gen 9, as well as farefetchd

**Review by Tempo-anon — CHANGES_REQUESTED** [review]
We really need to agree on how to deal with TM levelsets and what to do in cases where there's Pokemon that lose learning moves in between generations

**Comment by EmberCM** [comment]
> We really need to agree on how to deal with TM levelsets and what to do in cases where there's Pokemon that lose learning moves in between generations

The way I implemented it currently was giving it to all the Pokemon in the list no matter the generation or if they lost it.

**Comment by Fontbane** [comment]
> > We really need to agree on how to deal with TM levelsets and what to do in cases where there's Pokemon that lose learning moves in between generations
> 
> The way I implemented it currently was giving it to all the Pokemon in the list no matter the generation or if they lost it.

Yeah this is how we've been doing things so far, should be fine. Also maybe change from [Bug] to [Balance].

**Comment by Tempo-anon** [comment]
Too many merge conflicts... closing this PR for now

**Pull request closed without merging**